src/bin/download_sysext.rs

```diff
@@ -295,6 +295,10 @@ struct Args {
     #[argh(option, short = 'o')]
     output_dir: String,
 
+    /// target filename in directory
```

Comment:
Ok, the possibility matrix of command-line options is getting bigger and bigger. I am slightly concerned how big it is going to be in the end.

We should add a detailed help message for `--target-filename`, like `this option can be used only together with --take-first-match`.

Reply:
Good idea. It can also be used with `--payload-url`

Reply:
Updated it:
```
Options:
  -o, --output-dir  the directory to download the sysext images into
  -n, --target-filename
                    target filename in directory, requires --payload-url or
                    --take-first-match
```

Reply:
I've also removed one option here to keep the number of options low: https://github.com/flatcar/ue-rs/pull/50